urllib3/contrib/pyopenssl.py

```diff
@@ -138,7 +138,19 @@ def _dnsname_to_stdlib(name):
     then on Python 3 we also need to convert to unicode via UTF-8 (the stdlib
     uses PyUnicode_FromStringAndSize on it, which decodes via UTF-8).
     """
-    name = idna.encode(name)
+    def idna_encode(name):
```

Comment:
Can...can I impose on you to click the 'approve' button? I want all the green ticks. =P